packages/apps/graph-client/src/graphql/fields/signer.graph.ts

```diff
@@ -3,7 +3,10 @@ import type { DocumentNode } from 'graphql';
 
 export const CORE_SIGNER_KEY_FIELDS: DocumentNode = gql`
   fragment CoreSignerKeyFields on Signer {
-    capabilities
+    clist {
```

Comment:
Don't we want to call this `capabilitiesList`?

Reply:
I think here we want to respect the existing model which has this field called clist (please confirm @alber70g). You can check the model here: https://api.chainweb.com/openapi/pact.html#tag/model-payload